terraform/eval_read_data.go

```diff
@@ -38,7 +38,8 @@ func (n *EvalReadDataDiff) Eval(ctx EvalContext) (interface{}, error) {
 		provider := *n.Provider
 		config := *n.Config
 
-		diff, err := provider.ReadDataDiff(n.Info, config)
```

Comment:
Good catch. This is annoying feature of Go that the GitHub level of context on diffs don't really help with!